server/http/dialog/install.go

```diff
@@ -44,6 +44,15 @@ func NewInstallAppDialog(m *apps.Manifest, secret, pluginURL string, commandArgs
 			HelpText:    "TODO: How to obtain the App Secret",
 			Default:     secret,
 		})
+		elements = append(elements, model.DialogElement{
+			DisplayName: "App webhook secret:",
```

Comment:
Don't we need that for AWS apps?

Reply:
we do

Reply:
The Webhook Secret is now auto-generated if the App needs one.